doc/spec_TransferElement.dox

```diff
@@ -179,9 +179,14 @@ This document describes the behaviour of the TransferElement base class, the NDR
         - 8.5.1.2 Meta-backends like the LogicalNameMappingBackend delegate Device::activateAsyncRead() to all of their target backends, as they also delegate the creation of the TransferElements to their targets.
       - \anchor transferElement_B_8_5_2 8.5.2 In case the device does not send an initial value after the subscription, the accessor implementation must get an initial value synchronously and treat it as if it would have been received, i.e. push it to the queue. This must happen after the actual asynchronous sending has been turned on to make sure no update is missed. (*) [\ref UnifiedTest_TransferElement_B_8_5_2 "U"]
       - \anchor transferElement_B_8_5_3 8.5.3 If a transfer element is created while the device is opened, functional and  Device::activateAsyncRead() has been called, the asynchronous read is activated immediately (incl. sending of the initial value). This ensures that either all or none of the transfer elements are receiving asyncronously send data. [\ref UnifiedTest_TransferElement_B_8_5_3 "Needs to be changed, most probably testing the wrong thing. U"]
-      - 8.5.4 If Device::activateAsyncRead() is called while the device is not opened or has an error, this call has no effect. If it is called when no deactivated transfer element exists, this call also has no effect.
+      - \anchor transferElement_B_8_5_4 8.5.4 A Device::activateAsyncRead() call has no effect if
+         - 8.5.4.1 the device is not opened
+         - 8.5.4.2 the device has an error
+         - 8.5.4.3 no transfer elements without wait_for_new_data exist
```

Comment:
This point is wrong. It should have read
```no transfer elements with wait_for_new_data exist```, but that is also wrong, already according to this spec.
The minimum change is: The backend internally stores that activateAsyncRead has been called to fulfill B.8.5.3 when a new accessor with the flag is created, even if there are no such accessors yet. As all backends are in principle prepared for async accessors (the async domains container is already in the base classes) this conceptually always the case.